static/app/components/replays/header/replayMetaData.tsx

```diff
@@ -93,4 +94,16 @@ const KeyMetricData = styled('dd')`
   line-height: ${p => p.theme.text.lineHeightBody};
 `;
 
+const Count = styled('span')`
+  font-variant-numeric: tabular-nums;
+`;
+
+const ClickCount = styled(Count)<{color: ColorOrAlias}>`
+  color: ${p => p.theme[p.color]};
+  display: flex;
+  width: 40px;
```

Comment:
We shouldn't have this here. It'll do weird things when there are more than 2 digits in the number.

```suggestion
  width: 40px;
```